### ISSUE TITLE
Mismatch between Tiny-ImageNet and ImageNet Labels

### ISSUE THREAD

**Issue by LucaVergolani** [selimfirat/greenedge-energy-aware-classification#2]
I noticed that the Tiny-ImageNet labels are not in the range 0-199 nor in the range 0-999, this caused some problems when testing the dataset with a pretrained model on ImageNet, such as EfficientNet, since it seems like there is no direct relation between the two datasets labels. 
For this reason, I had a conversation with the Deep Lake staff today to ask how to correctly map the Tiny-ImageNet labels to the ImageNet labels. 
They informed me that by the end of the week, they will update the Deep Lake Tiny-ImageNet dataset to correct the labels so that they correspond to the actual ImageNet labels.

**Comment by selimfirat** [comment]
Thanks a lot for informing us!

Although the range is different, are there 200 distinct labels now?

It will not be straightforward to use pretrained models anyway since image resolutions are different. Even though the labels and resolution are different, pretrained models can still help, but you probably need to get your hands dirty.

This bug might be affecting the example in the notebook, though. We will check it out soon.

**Comment by selimfirat** [comment]
Btw, we encourage you to train your own models since accuracy is not the only factor. I believe it is hard to achieve good scores with these models alone anyway. The report will also matter, and I look forward to seeing smart ideas (such as early exit, clustering etc.) there!

**Comment by LucaVergolani** [comment]
> Thanks a lot for informing us!
> 
> Although the range is different, are there 200 distinct labels now?
> 
> It will not be straightforward to use pretrained models anyway since image resolutions are different. Even though the labels and resolution are different, pretrained models can still help, but you probably need to get your hands dirty.
> 
> This bug might be affecting the example in the notebook, though. We will check it out soon.

Yes, i personally checked and there are 200 classes even though the range is different.
